### PR TITLE
DB-15820: Revert behavior of XGStatement::setMaxRows() to silently omit excess rows (soft limit)

### PR DESCRIPTION
**Pull request by jwilliams-ocient**

### CODE REVIEW

**Comment by jwilliams-ocient** [comment]
ODBC changes => https://github.com/Xeograph/xgsrc/pull/16368

**Review by tli-ocient — APPROVED** [review]
version bump and release.adoc changes necessary

File: src/main/java/com/ocient/jdbc/XGConnection.java [hunk 1]
```diff
@@ -2770,15 +2770,15 @@ public void setHoldability(final int arg0) throws SQLException
 
 	public int setMaxRows(final Integer maxRows, final boolean reset)
 	{
+		// Set a "soft" limit on the number of rows returned. "Soft" in this case
+		// implies the query should silently omit excess rows
 		LOGGER.log(Level.INFO, String.format("Setting maxrow to: %d", maxRows));
-		this.maxRows = maxRows;
```
[the remark this answers is not in the document]
Comment (jwilliams-ocient) [reply src/main/java/com/ocient/jdbc/XGConnection.java:2774]: great suggestion

**Review by markwschmidt — APPROVED** [review]

**Comment by markwschmidt** [comment]
Are we waiting for anything else on this?

**Comment by jwilliams-ocient** [comment]
No, I just don’t have merge rights in this repo. @tli-ocient can you merge for me?

**Review by jasonar81 — APPROVED** [review]

**Pull request merged**